src/seedu/addressbook/commands/Command.java

```diff
@@ -65,4 +66,16 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   String getPrintableString(Printable... printables){
+	   String printableString = "";
+	   for(Printable p: printables) {
+		   printableString += p;
```

Comment:
since Printable does not implement `toString`, this would likely fail. you need to use the `getPrintableString()` method